lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -89,5 +99,13 @@ def order_permissions
     def report_line_items
       @report_line_items ||= Reports::LineItems.new(order_permissions, options)
     end
+
+    def report_variant_overrides
+      @report_variant_overrides ||=
+        Reports::VariantOverrides.new(
+          line_items: order_permissions.visible_line_items,
+          distributor_ids: report_line_items.orders.result.select('DISTINCT distributor_id'),
+        ).indexed
```

Comment:
Personally I dont like the keyword parameters, specially because they are both mandatory in this case. I'd just use normal parameters.

Reply:
I think the keywork parameters here are adding some readability, I think the problem here is the complexity, it should be extracted to a separate method, like this:
```ruby
    Reports::VariantOverrides.new(report_distributor_ids, variants_of_visible_line_items).indexed
   
    def variants_of_visible_line_items
        order_permissions.visible_line_items.select(:variant_id)
    end
    def report_distributor_ids
       report_line_items.orders.result.select('DISTINCT distributor_id')
    end
```

this is not so important but I thought of sharing.

Reply:
I've removed the keyword params, I personally like them for public methods just to make calls a little more readable, but very happy to stick with convention / preferences of the core team.

Reply:
yeah, it's a detail, the main problem here is really the complexity in building the values. A good rule of thumb is: one dot per line of code.